src/components/manifold-invoices/manifold-invoices.tsx

```diff
@@ -168,8 +271,7 @@ export class ManifoldInvoices {
 
   render() {
     return (
-      <div class="ManifoldInvoices">
-        <h1 class="ManifoldInvoices__Heading">Billing Statements</h1>
+      <div class="ManifoldInvoices" aria-live="polite">
```

Comment:
a11y 🙌

Reply:
Yeah it felt wrong to me that this component basically “navigates” without affecting the router (not that it should—that might mess things up for the provider). So I did the next best thing by alerting screen readers of updates